paasta_tools/cli/cmds/spark_run.py

```diff
@@ -1150,6 +1155,40 @@ def paasta_spark_run(args):
         needs_docker_cfg=needs_docker_cfg,
         auto_set_temporary_credentials_provider=auto_set_temporary_credentials_provider,
     )
+
+    # Mass enable DRA config
+    if (
+        args.cluster_manager == CLUSTER_MANAGER_K8S
+        and "spark.dynamicAllocation.enabled" not in spark_conf
+    ):
+        num_executors = int(spark_conf["spark.executor.instances"])
+        for exec_range in EXECUTOR_RANGES:
+            if exec_range[0] <= num_executors <= exec_range[1]:
+
+                log.warn(
+                    PaastaColors.yellow(
+                        "Spark Dynamic Resource Allocation (DRA) enabled for this batch. "
+                        "More info: y/spark-dra. If you wish to disable DRA, please provide "
```

Comment:
Add slightly more details.
If you think your job is performing worse than before, or your job is not catered towards dynamic instance changes, then <do xyz>.

Note that there needs to be an announcement to make sure, if any of the jobs are not catered towards dynamic allocation, they need to be disabled first, before rolling out this.

Reply:
Added the details